### PR TITLE
Fix installation documentation

### PR DESCRIPTION
**Pull request by adamwalach**
**Description**

Changes proposed in this pull request:

- added missing step

**Related issue(s)**
* #2403

### CODE REVIEW

File: docs/kyma/docs/037-inst-xip.io-domain.md [hunk 1]
```diff
@@ -67,8 +65,33 @@ Follow the respective instructions to deploy a cluster Kyma cluster with wildcar
   ```
 (cat installation/resources/installer.yaml ; echo "\n---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "\n---" ; cat installation/resources/installer-cr-cluster-xip-io.yaml.tpl) | sed -e "s/__EXTERNAL_PUBLIC_IP__/$EXTERNAL_PUBLIC_IP/g" | sed -e "s/__REMOTE_ENV_IP__/$CONNECTOR_IP/g" | sed -e "s/__APPLICATION_CONNECTOR_DOMAIN__/$CONNECTOR_IP.xip.io/g" | sed -e "s/__SKIP_SSL_VERIFY__/true/g" | sed -e "s/__.*__//g" > my-kyma.yaml
   ```
-3. Follow [these](#installation-install-kyma-on-a-gke-cluster-deploy-kyma) instructions to install Kyma using the configuration file you prepared.  
-
+### Kyma installation
+
+You can either choose the pre-build image of the Kyma Installer or build your own.
```
Comment (tomekpapiernik) [inline docs/kyma/docs/037-inst-xip.io-domain.md]: `pre-build image` -> I think you need an adjective here, right? 
if so -> `prebuilt`
[CLICK FOR REF](https://en.wiktionary.org/wiki/prebuilt)

File: docs/kyma/docs/037-inst-xip.io-domain.md [hunk 2]
```diff
@@ -67,8 +65,33 @@ Follow the respective instructions to deploy a cluster Kyma cluster with wildcar
...
+  1. Build an image that is based on the current Installer image and includes the current installation and resources charts. Run:
+     ```
+     docker build -t kyma-installer:latest -f tools/kyma-installer/kyma.Dockerfile . --build-arg INSTALLER_VERSION=63484523
+     ```
+  2. Push the image to your Docker Hub:
+     ```
+     docker tag kyma-installer:latest [YOUR_DOCKER_LOGIN]/kyma-installer:latest
+     docker push [YOUR_DOCKER_LOGIN]/kyma-installer:latest
+     ```
+* To use a prebuild image, go to [this](https://github.com/kyma-project/kyma/releases/) page and check the version of the latest release. Your URL looks as follows:
```
Comment (tomekpapiernik) [inline docs/kyma/docs/037-inst-xip.io-domain.md]: prebuild -> `prebuilt`

Comment (tomekpapiernik) [inline docs/kyma/docs/037-inst-xip.io-domain.md]: Go to the page and check the version, then you get a link? I don't think so. 
`Go to the page -> choose the version you want to use -> grab the link`

File: docs/kyma/docs/037-inst-xip.io-domain.md [hunk 3]
```diff
@@ -67,8 +65,33 @@ Follow the respective instructions to deploy a cluster Kyma cluster with wildcar
...
+```eu.gcr.io/kyma-project/kyma-installer:{latest version}```
```
Comment (tomekpapiernik) [inline docs/kyma/docs/037-inst-xip.io-domain.md]: `{latest version}` -> do I have to use the latest version? Can't I use any version compatible with XIP? (0.6 and up?)

Comment (tomekpapiernik) [inline docs/kyma/docs/037-inst-xip.io-domain.md]: **Wrong formatting**
Additionally, when using three 'code ticks', place them in lines above and below the text you want to format as a code block.

File: docs/kyma/docs/037-inst-xip.io-domain.md [hunk 4]
```diff
@@ -67,8 +65,33 @@ Follow the respective instructions to deploy a cluster Kyma cluster with wildcar
   ```
 (cat installation/resources/installer.yaml ; echo "\n---" ; cat installation/resources/installer-config-cluster.yaml.tpl ; echo "\n---" ; cat installation/resources/installer-cr-cluster-xip-io.yaml.tpl) | sed -e "s/__EXTERNAL_PUBLIC_IP__/$EXTERNAL_PUBLIC_IP/g" | sed -e "s/__REMOTE_ENV_IP__/$CONNECTOR_IP/g" | sed -e "s/__APPLICATION_CONNECTOR_DOMAIN__/$CONNECTOR_IP.xip.io/g" | sed -e "s/__SKIP_SSL_VERIFY__/true/g" | sed -e "s/__.*__//g" > my-kyma.yaml
   ```
-3. Follow [these](#installation-install-kyma-on-a-gke-cluster-deploy-kyma) instructions to install Kyma using the configuration file you prepared.  
-
+### Kyma installation
+
+You can either choose the pre-build image of the Kyma Installer or build your own.
+
+* To build your own image:
```
Comment (tomekpapiernik) [inline docs/kyma/docs/037-inst-xip.io-domain.md]: The formatting of this list renders with roman numerals. We don't use this kind of formatting. Please switch to a format used in other documents.

File: docs/kyma/docs/037-inst-xip.io-domain.md (hunk 3 above)
Comment (adamwalach) [reply docs/kyma/docs/037-inst-xip.io-domain.md]: I changed it to single 'code ticks'

Comment (adamwalach) [reply docs/kyma/docs/037-inst-xip.io-domain.md]: I added Note

File: docs/kyma/docs/037-inst-xip.io-domain.md (hunk 2 above)
Comment (adamwalach) [reply docs/kyma/docs/037-inst-xip.io-domain.md]: The intention is to take only the version string and put it in the URL. I modified the description to make it clear

File: docs/kyma/docs/04-06-gke-xip.io-domain.md [hunk 5]
```diff
@@ -67,8 +65,36 @@ Follow the respective instructions to deploy a cluster Kyma cluster with wildcar
...
+
+* To use a prebuilt image, go to [this](https://github.com/kyma-project/kyma/releases/) page and check the available releases. You can chose version 0.6 or higher.
+ Put the chosen release version in the following URL:
+`eu.gcr.io/kyma-project/kyma-installer:{RELEASE_VERSION}`
+
+* To build your own Installer image from sources, run:
+  ```
+  docker build -t kyma-installer:latest -f tools/kyma-installer/kyma.Dockerfile .
+  ```
+  Create a new `kyma-installer` public repository on [Docker Hub](https://hub.docker.com/)  and push the image:
```
Comment (klaudiagrz) [inline docs/kyma/docs/04-06-gke-xip.io-domain.md]:
```suggestion
  Create a new `kyma-installer` public repository on [Docker Hub](https://hub.docker.com/) and push the image:
```

File: docs/kyma/docs/04-07-aks-xip.io-domain.md [hunk 6]
```diff
@@ -68,20 +68,22 @@ Use this command to prepare a configuration file that deploys Kyma with [`xip.io
...
+
+* To use a prebuilt image, go to [this](https://github.com/kyma-project/kyma/releases/) page and check the available releases. You can chose version 0.6 or higher.
+ Put the chosen release version in the following URL:
+`eu.gcr.io/kyma-project/kyma-installer:{RELEASE_VERSION}`
+
+* To build your own Installer image from sources, run:
+  ```
+  docker build -t kyma-installer:latest -f tools/kyma-installer/kyma.Dockerfile .
+  ```
+  Create a new `kyma-installer` public repository on [Docker Hub](https://hub.docker.com/)  and push the image:
```
Comment (klaudiagrz) [inline docs/kyma/docs/04-07-aks-xip.io-domain.md]:
```suggestion
  Create a new `kyma-installer` public repository on [Docker Hub](https://hub.docker.com/) and push the image:
```

**Pull request merged**